stdlib/Test/src/Test.jl

```diff
@@ -505,7 +498,7 @@ function do_test(result::ExecutionResult, orig_expr)
         value = result.value
         testres = if isa(value, Bool)
             # a true value Passes
-            value ? Pass(:test, nothing, nothing, value) :
+            value ? Pass(:test, orig_expr, result.data, value) :
```

Comment:
I wonder if we would also like to store the `result.source` in `Pass`.
Since presumably the justification for why you would not want it is that it wasn't a failure.
But that is also a justification for why you wouldn't want the `orig_expr` and the `result.data`.
Which we do infact want.

Reply:
Yes good point, I'll add that in